change/@ni-nimble-blazor-1d363eda-276d-4735-9e97-20d631d96957.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Was discussed at stand-up to explore the cost of using anchored-region for this use-case:

Anchored Region

Pros:
- Aligns with our existing pattern across controls (menu, menu-button, tooltip, is that all?)
- We aren't mixing popover implementations across components
- Has some more usage in the wild, i.e. the floating-ui library is part of FAST beta that hasn't released yet

Cons:
- Requires template forking
- Anchored region was proprietary to FAST and is deprecated, have to switch eventually.

If it's true that only a few of our controls use anchored region and two of three are already templated owned by us (tooltip and menu-button), then maybe we should be looking at the cost of migrating to floating ui for all of them